src/System.Private.CoreLib/shared/System/Diagnostics/SymbolStore/ISymDocumentWriter.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics.SymbolStore
+{
+    public interface ISymbolDocumentWriter
+    {
```

Comment:
I would have expected the file to be named the same as the interface. I know it was wrong before, but maybe it's a chance to fix it.

Also, did you intentionally remove the comments?